crates/solver/src/settlement/settlement_encoder.rs

```diff
@@ -52,6 +52,13 @@ pub struct SettlementEncoder {
     unwraps: Vec<UnwrapWethInteraction>,
 }
 
+/// An encoded trade.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub enum EncoderTrade<'a> {
```

Comment:
nit: The comment and the name of the enum seem to disagree.
The enum name appears to be more appropriate than the comment (the trade is actually not encoded yet)

Reply:
Fixed the comment.